### PR TITLE
Setup basic iOS CI

### PR DESCRIPTION
**Pull request by irwansetiawan**
Basic iOS CI to verify that the project builds

### CODE REVIEW

**Comment by irwansetiawan** [comment]
I'm not sure how to build "all schemes" in a workspace, doesn't seem like possible from my quick search. 

Currently, we need to update the build script for every new schemes.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  test:
```
Comment (vguerci) [inline .github/workflows/main.yml]: OPT: Maybe call this "build" as it does no tests ;)

**Review by vguerci — DISMISSED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (irwansetiawan) [reply .github/workflows/main.yml]: Make sense! Edited the job name.

**Comment by irwansetiawan** [comment]
- Edited job name to `build`
- Removed iOS version on the build parameter (default to latest) that may cause build failure when available iOS version is updated

**Review by TurpIF — APPROVED** [review]

**Pull request merged**